### PR TITLE
Improve components and fixed map style

### PR DESCRIPTION
**Pull request by aaranadev**
Improve the components with new components and utils.

Also the map style is fixed.

### CODE REVIEW

File: template-skeleton/template/src/components/views/Main.js [hunk 1]
```diff
@@ -205,24 +203,15 @@ const useStylesMapContainer = makeStyles((theme) => ({
...
     zIndex: 1,
 
-    [theme.breakpoints.down('sm')]: {
-      bottom: theme.spacing(4),
-      left: theme.spacing(2),
+    [theme.breakpoints.down('xs')]: {
+      display: 'none',
     },
   },
   cartoLogoMap: {
```
Comment (alasarr) [inline template-skeleton/template/src/components/views/Main.js:210]: Why is this class defined here? Don't be better to move this to CartoLogoMap component?

**Review by alasarr — CHANGES_REQUESTED** [review]
Please, test carefully it works with the different error messages.

Update changelog.md

File: template-skeleton/template/src/components/views/Main.js (hunk 1 above)
Comment (aaranadev) [reply template-skeleton/template/src/components/views/Main.js:210]: It's not CartoLogoMap, it's ZoomControl. It's always better define if a children is shown or not from parent, so the children haven't the context. This code is in sample-app.

Comment (aaranadev) [reply template-skeleton/template/src/components/views/Main.js:214]: Done

**Review by VictorVelarde — APPROVED** [review]
🚀

**Pull request merged**